gaphor/transaction.py

```diff
@@ -125,11 +125,11 @@ def _close(self):
 
     def _handle(self, event):
         try:
-            component_registry = self.component_registry
+            event_manager = self.event_manager
         except (application.NotInitializedError, application.ComponentLookupError):
             log.warning("Could not lookup component_registry. Not emitting events.")
```

Comment:
Should this log also be renamed to event_manager?

Reply:
Ah, nice catch 😄